### PR TITLE
feat: add first draft of german translation.

### PR DESCRIPTION
**Pull request by franziska-warncke**
Added a first draft of german translation. 

What is still needed: 
- A writer review to make sure the translation went well ✅ 
- Translation of the images  ✅

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx [hunk 1]
```diff
@@ -0,0 +1,287 @@
+---
+sidebar_position: 5
+title: Carbon Awareness
```
Comment (russelltrow) [inline i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx:3]: Should this title be translated?

**Review by russelltrow — COMMENTED** [review]
Translations looks great and I've tested locally and it works perfectly.

**Review by cdhaens — CHANGES_REQUESTED** [review]
See inline

**Comment by russelltrow** [comment]
Hey @franziska-warncke when might you get chance to review @cdhaens feedback please?

File: i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+## Zusammenfassung
+
+- Die Entwicklung einer energieeffizienten Anwendung ist also gleichbedeutend mit der Entwicklung einer kohlenstoffeffizienten Anwendung.
+- Grüne Software übernimmt die Verantwortung für ihren Stromverbrauch und ist so konzipiert, dass sie so wenig wie möglich verbraucht.
+- Die Quantifizierung des Energieverbrauchs einer Anwendung ist ein Schritt in die richtige Richtung, um zu überlegen, wie eine Anwendung effizienter arbeiten kann. Das Verständnis des Energieverbrauchs Ihrer Anwendung ist jedoch nicht die einzige Geschichte. Die Hardware, auf der Ihre Software läuft, verbraucht einen Teil des Stroms für betriebliche Gemeinkosten. Dies wird im Bereich der Cloud als Stromverbrauchseffizienz (PUE) bezeichnet.
```
Comment (tamarau) [inline i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx:89]:
```suggestion
- Die Quantifizierung des Energieverbrauchs einer Anwendung ist ein Schritt in die richtige Richtung, um zu überlegen, wie eine Anwendung effizienter arbeiten kann. Das Verständnis des Energieverbrauchs Ihrer Anwendung ist jedoch nicht die einzige Geschichte. Die Hardware, auf der Ihre Software läuft, verbraucht einen Teil des Stroms für betriebliche Gemeinkosten. Dies wird im Bereich der Cloud als Stromverbrauchseffizienz (PUE - Power Usage Effectiveness) bezeichnet.
```

File: i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+          isCorrect: false,
+        },
+        {
+          text: "Joule",
+          isCorrect: true,
+        },
+      ],
+    },
+    {
+      question: "elche Anwendung verbraucht die meiste Energie?",
```
Comment (tamarau) [inline i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx:181]:
```suggestion
      question: "Welche Anwendung verbraucht die meiste Energie?",
```

File: i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+      answers: [
+         {
+          text: "Eine Anwendung, die 20 kWh in einem Rechenzentrum mit einem PUE-Wert von 1 verbraucht",
+          isCorrect: true,
+        },
+        {
+          text: "Eine Anwendung, die 15kWh in einem Rechenzentrum mit einer PUE von 1,2 verbraucht",
```
Comment (tamarau) [inline i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx:188]:
```suggestion
          text: "Eine Anwendung, die 15 kWh in einem Rechenzentrum mit einer PUE von 1,2 verbraucht",
```

File: i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+          isCorrect: false,
+        },
+        {
+          text: "Eine Anwendung, die 10kWh in einem Rechenzentrum mit einem PUE-Wert von 1,5 verbraucht",
```
Comment (tamarau) [inline i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx:192]:
```suggestion
          text: "Eine Anwendung, die 10 kWh in einem Rechenzentrum mit einem PUE-Wert von 1,5 verbraucht",
```

File: i18n/de/docusaurus-theme-classic/navbar.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  "title": {
+    "message": "Green Software for Practitioners",
+    "description": "Der Titel in der Navigationsleiste"
+  },
+  "item.label.Take the exam": {
+    "message": "Mach die Zertifizierung Green Software for Practitioners (LFC131) ",
+    "description": "Navbar-Element mit Beschriftung"
+  },
+  "item.label.a Green Software Foundation project": {
+    "message": "ein Green Software Foundation Projekt",
```
Comment (tamarau) [inline i18n/de/docusaurus-theme-classic/navbar.json:11]:
```suggestion
    "message": "Ein Green Software Foundation Projekt",
```

**Review by tamarau — COMMENTED** [review]
Hut ab :) Ein paar Kleinigkeiten meinerseits

**Comment by franziska-warncke** [comment]
Sorry, I was on vacation and after that I got sick. I will look into this now. :)

File: i18n/de/docusaurus-plugin-content-docs/current/carbon-efficiency.mdx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+Das Verständnis der Auswirkungen von Treibhausgasen auf unsere Umwelt ist der Schlüssel zum Verständnis des eigenen Kohlenstoff-Fußabdrucks der Software. Sie lernen die verschiedenen Arten von Treibhausgasen kennen, die in der Umwelt vorhanden sind, wie sie emittiert und gemessen werden und was von verschiedenen Organisationen auf der ganzen Welt getan wird, um diese Emissionen zu kontrollieren und zu reduzieren.
+
+Sie erfahren etwas über das Treibhausgasprotokoll und was es für Praktiker der grünen Software bedeutet.
+
+## Schlüsselkonzepte
+
+### Globale Erwärmung vs. Klimawandel
+
+Die [globale Erwärmung](https://climate.nasa.gov/global-warming-vs-climate-change/) ist die seit der vorindustriellen Zeit (zwischen 1850 und 1900) beobachtete langfristige Erwärmung des Klimasystems der Erde, die auf menschliche Aktivitäten, vor allem die Verbrennung fossiler Brennstoffe, zurückzuführen ist. Unter [Klimawandel](https://climate.nasa.gov/global-warming-vs-climate-change/) versteht man langfristige Veränderungen der Temperaturen und Wettermuster. Diese Verschiebungen können natürlich sein, aber seit den 1800er Jahren sind menschliche Aktivitäten die Hauptursache für den Klimawandel.
```
[the remark this answers is not in the document]
Comment (franziska-warncke) [reply i18n/de/docusaurus-plugin-content-docs/current/carbon-efficiency.mdx:28]: Mir hat ....die auf menschengemachte Aktivitäten.... besser gefallen

File: i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+
+![alt_text](./images/11_marginal_CI.png "image_tooltip")
+
+Wenn Sie den Stromverbrauch drosseln, bedeutet das, dass Sie überschüssige Energie haben, die Sie abgeben könnten. Erneuerbare Energie ist bereits die billigste, so dass die gedrosselte erneuerbare Energie die billigste einspeisbare Energiequelle sein wird. Die Kraftwerke für erneuerbare Energien werden dann die Energie verkaufen, die sie sonst hätten abnehmen müssen.
+
+2. **Mehr Energie aus fossilen Kraftwerken kaufen**.
+
+![alt_text](./images/12_marginal_CI.png "image_tooltip")
+
+Fossile Brennstoffe sind von Natur aus abschaltbar; sie können die Energieproduktion schnell erhöhen, indem sie mehr verbrennen. Allerdings kostet Kohle Geld, so dass dies die am wenigsten bevorzugte Lösung ist.
```
[the remark this answers is not in the document]
Comment (franziska-warncke) [reply i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx:92]: nice catch! 👍 Das wusste ich nicht.

File: i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+
+Die Energiemärkte gehören zu den komplexesten Märkten der Welt, so dass die obige Erklärung eine Vereinfachung darstellt. Es ist jedoch wichtig zu verstehen, dass unser Ziel darin besteht, Investitionen in kohlenstoffärmere Energiequellen wie erneuerbare Energien zu erhöhen und Investitionen in kohlenstoffreichere Quellen wie Kohle zu verringern. Der beste Weg, um sicherzustellen, dass das Geld in die richtige Richtung fließt, ist, dafür zu sorgen, dass Sie Strom mit der geringsten Kohlenstoffintensität verwenden.
+
+## Wie man klimabewusster ist
+
+:::tip
+
+Strom zu nutzen, wenn die Kohlenstoffintensität gering ist, ist der beste Weg, um sicherzustellen, dass Investitionen in Anlagen mit geringen Kohlenstoffemissionen fließen und nicht in Anlagen mit hohen Kohlenstoffemissionen.
```
[the remark this answers is not in the document]
Comment (franziska-warncke) [reply i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx:100]: Viel besser! Danke 🌻

File: i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+        },
+        { text: "0gCO2eq/kWh", isCorrect: false },
+      ],
+    },
+    {
+      question: "Was ist die Standardeinheit für die Kohlenstoffintensität?",
+      answers: [
+        { text: "gCO2eq/kWh", isCorrect: false },
+        { text: "gCO2e/kWh", isCorrect: false },
+        { text: "Eines der oben genannten", isCorrect: true },
```
[the remark this answers is not in the document]
Comment (franziska-warncke) [reply i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx:195]: Agree! :)

**Comment by franziska-warncke** [comment]
Can you please have a look again. Thanks 🌻

**Comment by russelltrow** [comment]
Hey @franziska-warncke are you happy for us to merge this and then pick-up any further changes later?

I'm aware you've been working on this for a while now.

**Comment by franziska-warncke** [comment]
@russelltrow Let me add the pictures, and then it is ready for merging! :) 
I will quickly do that :)

**Comment by franziska-warncke** [comment]
Done! It is now ready to be merged :)

**Review by cdhaens — APPROVED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx [hunk 11]
```diff
@@ -0,0 +1,287 @@
...
+
+import Quiz from "/src/components/Quiz";
+
+:::note
+Dies ist eine gemeinschaftlich erstellte Übersetzung. Sie hat nur begrenzte Unterstützung und entspricht möglicherweise nicht der neuesten deutschen Version des Kurses.
+:::
+
+:::tip Prinzip
+
+_Tun Sie mehr, wenn der Strom sauberer ist, und tun Sie weniger, wenn der Strom schmutziger ist._
```
[the remark this answers is not in the document]
Comment (franziska-warncke) [reply i18n/de/docusaurus-plugin-content-docs/current/carbon-awareness.mdx:14]: Werde ich im nächsten PR dann umsetzen :)

File: i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+
+### Energieproportionalität
+
+Die [Energieproportionalität] (https://research.google/pubs/pub33387/), die erstmals 2007 von Google-Ingenieuren vorgeschlagen wurde, misst **das Verhältnis zwischen der von einem Computer verbrauchten Energie und der Rate, mit der nützliche Arbeit geleistet wird** (seine Auslastung).
+
+Die Auslastung misst, wie stark die Ressourcen eines Computers genutzt werden, und wird in der Regel als Prozentsatz angegeben. Ein voll ausgelasteter Computer, der mit seiner maximalen Kapazität läuft, hat einen hohen Prozentsatz, während ein Computer im Leerlauf, der nicht ausgelastet ist, einen niedrigeren Prozentsatz hat.
+
+Das Verhältnis zwischen Leistung und Auslastung ist nicht proportional. Mathematisch gesehen bedeutet Proportionalität zwischen zwei Variablen, dass ihre Verhältnisse gleichwertig sind. Bei 0 % Auslastung kann ein Computer beispielsweise 100 W verbrauchen, bei 50 % sind es 180 W und bei 100 % 200 W. Die Beziehung zwischen Stromverbrauch und Auslastung ist nicht linear und geht nicht durch den Ursprung.
+
+![alt_text](./images/06_energy_proportionality_updated.png "image_tooltip")
```
Comment (russelltrow) [inline i18n/de/docusaurus-plugin-content-docs/current/energy-efficiency.mdx:75]: Need to remove the "_updated" from the filename

File: i18n/de/docusaurus-plugin-content-docs/current/introduction.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+Ein Green Software-[**Pattern**] (https://patterns.greensoftware.foundation/) ist ein konkretes Beispiel für die Anwendung eines oder mehrerer Prinzipien in einem realen Beispiel. Während Prinzipien die Theorie beschreiben, die grüner Software zugrunde liegt, sind Patterns die praktischen Ratschläge, die Software-Anwender heute in ihren Software-Anwendungen verwenden können. Patterns sind herstellerunabhängig.
+
+Eine Green Software **Praktik** ist ein Muster, das auf ein bestimmtes Produkt eines Anbieters angewandt wird und die Anwender darüber informiert, wie sie dieses Produkt nachhaltiger nutzen können. 
+
+Praktiken sollten sich auf Muster beziehen, die sich auf Prinzipien beziehen sollten.
+
+Die Green Software Foundation veröffentlicht auch einen [Katalog anbieterneutraler grüner Softwaremuster] (https://patterns.greensoftware.foundation/) in verschiedenen Kategorien.
+ 
+![Green Software Principles, Patterns, and Practices](./images/GSF_Principles_Patterns_Practices_v2.png "Green Software Principles, Patterns, and Practices")
```
Comment (russelltrow) [inline i18n/de/docusaurus-plugin-content-docs/current/introduction.md:51]: This image appears to be missing from the Images folder

**Review by russelltrow — CHANGES_REQUESTED** [review]
Just need to fix a couple of broken images

**Review by russelltrow — APPROVED** [review]

**Pull request merged**